### PR TITLE
New post: GeyserMC

### PR DESCRIPTION
**Pull request by Mocab**
Could someone check if you can use extra ports? I dont have bedrock

### CODE REVIEW

File: _posts/minecraft/plugins/2021-08-08-geysermc.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "GeyserMC"
+categories: Minecraft
+tags: Plugins
+permalink: /minecraft/plugins/geysermc/
+---
+
+<div class="install-plugin">
+    <img style="border-radius: 7px;" src="https://raw.githubusercontent.com/FalixNodes-Software/help-center/main/assets/images/posts/plugins/geysermc/plugin.png">
```
Comment (itsjuoum) [inline _posts/minecraft/plugins/2021-08-08-geysermc.md:10]: The path for this image should be `../../../assets/images/posts/plugins/geysermc/plugin.png`. The same things goes for all images throughout the article.

File: _posts/minecraft/plugins/2021-08-08-geysermc.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    <p>GeyserMC</p>
+    <a href="https://ci.opencollab.dev//job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/target/Geyser-Spigot.jar">Download this Plugin</a>
```
Comment (itsjuoum) [inline _posts/minecraft/plugins/2021-08-08-geysermc.md:12]: I would change the download link to `https://ci.opencollab.dev//job/GeyserMC/job/Geyser/job/master/` as some people don't have spigot as their server software.

File: _posts/minecraft/plugins/2021-08-08-geysermc.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+</div>
+
+# How to make a cross-platform server
+
+## What is GeyserMC?
+Geyser is a proxy, bridging the gap between Minecraft: Bedrock Edition and Minecraft: Java Edition servers. The ultimate goal of this project is to allow Minecraft: Bedrock Edition users to join Minecraft: Java Edition servers as seamlessly as possible to allow for true cross-platform.
```
Comment (itsjuoum) [inline _posts/minecraft/plugins/2021-08-08-geysermc.md:18]: This entire text was copied from `https://geysermc.org/`. Please create your own description for GeyserMC, as we don't want trouble for plagiarizing.

File: _posts/minecraft/plugins/2021-08-08-geysermc.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+## How does it work?
+Geyser is a middleware, which translates all the incoming and outgoing packets. With this being said, Geyser works as a standalone proxy, meaning you can use it to join any modern Minecraft Java server. Geyser can also be installed on your server as a plugin, so you can easily make your server compatible with Minecraft: Bedrock Edition.
```
Comment (itsjuoum) [inline _posts/minecraft/plugins/2021-08-08-geysermc.md:21]: This entire text was also copied from `https://geysermc.org/`. Please create your own explanation for how GeyserMC works, as we don't want trouble for plagiarizing.

File: _posts/minecraft/plugins/2021-08-08-geysermc.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+## Remote:
+
+|Variable               |Description                                                        |
+|-----------------------|-------------------------------------------------------------------|
+|address:               |Leave this as "auto"                                               |
+|-----------------------|-------------------------------------------------------------------|
+|port:                  |Replace this with your server's port, you can get your port from the "networks" tab in the game panel. |
+|-----------------------|-------------------------------------------------------------------|
+|auth-type:             |Authentication type. Can be offline, online, or [floodgate](https://github.com/GeyserMC/Geyser/wiki/Floodgate), choose "offline" if you want cracked accounts (Tlauncher) to join your server.
```
Comment (itsjuoum) [inline _posts/minecraft/plugins/2021-08-08-geysermc.md:60]: I also think that you shouldn't mention other Minecraft clients that allow users to play Minecraft for free. We do not support piracy.

File: _posts/minecraft/plugins/2021-08-08-geysermc.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+# How to make a cross-platform server
+
+## What is GeyserMC?
+Geyser is a proxy, bridging the gap between Minecraft: Bedrock Edition and Minecraft: Java Edition servers. The ultimate goal of this project is to allow Minecraft: Bedrock Edition users to join Minecraft: Java Edition servers as seamlessly as possible to allow for true cross-platform.
+
+## How does it work?
+Geyser is a middleware, which translates all the incoming and outgoing packets. With this being said, Geyser works as a standalone proxy, meaning you can use it to join any modern Minecraft Java server. Geyser can also be installed on your server as a plugin, so you can easily make your server compatible with Minecraft: Bedrock Edition.
+
+## Prerequisites
+```
```
Comment (itsjuoum) [inline _posts/minecraft/plugins/2021-08-08-geysermc.md:24]: I believe the prerequisites would look better in an unordered list. You can achieve this in markdown by using "-".

**Review by itsjuoum — CHANGES_REQUESTED** [review]

File: _posts/minecraft/plugins/2021-08-08-geysermc.md (hunk 2 above)
Comment (Mocab) [reply _posts/minecraft/plugins/2021-08-08-geysermc.md:12]: This article is mainly for spigot and it's forks, it doesn't matter if it has spigot in its name it works for paper, purpur and other forks aswell

File: _posts/minecraft/plugins/2021-08-08-geysermc.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+## Bedrock:
+
+|Variable               |Description                                                        |
+|-----------------------|-------------------------------------------------------------------|
+|address:               |Leave it as 0.0.0.0                                                |
+|-----------------------|-------------------------------------------------------------------|
+|port:                  |Replace this with your server's port, you can get your port from the "networks" tab in the game panel. |
+
+![Image](https://raw.githubusercontent.com/FalixNodes-Software/help-center/main/assets/images/posts/plugins/geysermc/1.png)
```
Comment (DaddyChimken) [inline _posts/minecraft/plugins/2021-08-08-geysermc.md:50]: https://github.com/FalixNodes-Software/help-center/pull/38#discussion_r684782595

File: _posts/minecraft/plugins/2021-08-08-geysermc.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "GeyserMC"
+categories: Minecraft
+tags: Plugins
```
Comment (KorbsStudio) [inline _posts/minecraft/plugins/2021-08-08-geysermc.md:5]: This needs to be started with a lowercase 'p', please change `Plugins` to `plugins`.

Comment (Mocab) [reply _posts/minecraft/plugins/2021-08-08-geysermc.md:5]: image

in the README.md file it shows a capital letter, please change it to a lower case letter.

Comment (KorbsStudio) [reply _posts/minecraft/plugins/2021-08-08-geysermc.md]: The article won't show up at all in the help center if you don't use a lowercase p.

Comment (Mocab) [reply _posts/minecraft/plugins/2021-08-08-geysermc.md]: I am talking about this https://github.com/FalixNodes-Software/help-center#%EF%B8%8F-metadata

Comment (KorbsStudio) [reply _posts/minecraft/plugins/2021-08-08-geysermc.md]: https://github.com/FalixNodes-Software/help-center/blob/main/_posts/minecraft/plugins/2021-06-25-tabs.md#:~:text=Minecraft-,plugins,-/minecraft/plugins/tabs

File: _posts/minecraft/plugins/2021-08-08-geysermc.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+
+# Setting up GeyserMC
+
+1. Set up the server, you can use spigot or any of its forks, we recommend using Purpur or Paper.
+2. Open the file manager.
+3. Find and open a folder called "plugins".
+4. Upload the plugin you downloaded, you can find the download link at the top of this page.
+5. Restart the server.
+6. Go back to the plugins folder, you should see that a new folder was generated, it is usually called "Geyser-Spigot".
```
Comment (DaddyChimken) [inline _posts/minecraft/plugins/2021-08-08-geysermc.md:36]:
```suggestion
6. Go back to the "plugins" folder, you should see that a new folder was generated, it is usually named "Geyser-Spigot".
```

File: _posts/minecraft/plugins/2021-08-08-geysermc.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+7. Open it then click on "config.yml".
```
Comment (DaddyChimken) [inline _posts/minecraft/plugins/2021-08-08-geysermc.md:37]:
```suggestion
7. Open the folder and click on "config.yml".
```

File: _posts/minecraft/plugins/2021-08-08-geysermc.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+|-----------------------|-------------------------------------------------------------------|
+|port:                  |Replace this with your server's port, you can get your port from the "networks" tab in the game panel. |
+|-----------------------|-------------------------------------------------------------------|
+|auth-type:             |Authentication type. Can be offline, online, or [floodgate](https://github.com/GeyserMC/Geyser/wiki/Floodgate), choose "offline" if you want cracked accounts to join your server.
+
+![Image](../../../assets/images/posts/plugins/geysermc/2.png)
+
+
+9. Click on "Save" once you are done.
+10. Restart your server.
```
Comment (DaddyChimken) [inline _posts/minecraft/plugins/2021-08-08-geysermc.md:65]: It was always "the server", so to stay consistent, either change everything in this file to "your server" or keep "the server".
```suggestion
10. Restart the server.
```

File: _posts/minecraft/plugins/2021-08-08-geysermc.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+11. Done! you have successfully set up GeyserMC!
```
Comment (DaddyChimken) [inline _posts/minecraft/plugins/2021-08-08-geysermc.md:66]:
```suggestion
11. Done! You have successfully set up GeyserMC!
```

**Review by DaddyChimken — CHANGES_REQUESTED** [review]
:D

**Review by DaddyChimken — APPROVED** [review]
lgtm 👍

**Pull request merged**